app/services/validators/validator.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
rename to something other than validator, as requiring both 'validator' and './validator' will get confusing